### ISSUE TITLE
[Feature Request] Add to F-Droid

### ISSUE THREAD

**Issue by Pablohn26** [mmazzarolo/top-github#4]
I do not know if you know [F-Droid](https://f-droid.org/) _store_ , but would be awesome to have this app on that repository.  

Here you can check [Inclusion_How-To](https://f-droid.org/wiki/page/Inclusion_How-To) and [Inclusion_Policy](https://f-droid.org/wiki/page/Inclusion_Policy).

**Comment by mmazzarolo** [comment]
Yup, I know about F-Droid but I never thought about releasing this project on it.  
Indeed, it might be worthy take a look at it !
